src/TrackerCouncil.Smz3.Data/Configuration/ConfigTypes/HintsConfig.cs

```diff
@@ -173,12 +173,21 @@ public class HintsConfig : IMergeable<HintsConfig>
     /// <summary>
     /// Gets the hints for locations that don't have anything useful.
     /// <c>{0}</c> is placeholder for the name of the location.
+    /// <c>{1}</c> is placeholder for the character name.
```

Comment:
Might be worth adding it's the placeholder for the character that the item at that location is for (as opposed to the character that needs to pick it up)